### PR TITLE
Refactor: Move Donation Goal settings to the design tab

### PR DESCRIPTION
**Pull request by kjohnson**
## Description

- Co-locates the Donation Goal settings to the Design Preview tab (in addition to the Form Settings tab).
- Renames Goal Format to  Goal Type.
- Removes `percentage-raised` as a Goal Type

## Visuals

image

### CODE REVIEW

File: packages/form-builder/src/App.scss [hunk 1]
```diff
@@ -1,3 +1,5 @@
+@import 'styles/settings.scss';
```
Comment (jonwaldstein) [inline packages/form-builder/src/App.scss:1]: fyi, I moved these styles into a new file.  This stylesheet is getting a little crazy - let's make it a point to start organizing them and take advantage of scss

File: packages/form-builder/src/settings/donation-goal/index.js [hunk 2]
```diff
@@ -79,6 +71,19 @@ const DonationGoalSettings = () => {
                             onChange={(goalFormat) => dispatch(setFormSettings({goalFormat}))}
                         />
                     </PanelRow>
+                    <PanelRow>
+                        {/*
+                            @todo Do we need to reset the goal amount when the goal format changes?
+                                The context can be wildly different, ie Amount Raised vs Number of Donors.
+                                Perhaps we could implement smart default amounts for each goal format?
```
Comment (jonwaldstein) [inline packages/form-builder/src/settings/donation-goal/index.js:78]: Good thoughts, we should discuss this further.  I think some simple conditional logic will be a fine start for the `goalAmount` and description based on the goal format

File: packages/form-builder/src/settings/donation-goal/index.js [hunk 3]
```diff
@@ -19,18 +19,18 @@ const DonationGoalSettings = () => {
...
-        {
-            value: 'number-donations',
-            label: __('Number of Donations', 'give'),
-        },
-        {
-            value: 'number-donors',
-            label: __('Number of Donors', 'give'),
-        },
+        // {
+        //     value: 'percentage-raised',
```
Comment (jonwaldstein) [inline packages/form-builder/src/settings/donation-goal/index.js:23]: I'm fine with commenting these out for now but let's add a comment about our intentions here

Comment (kjohnson) [reply packages/form-builder/src/settings/donation-goal/index.js:24]: I added a `@todo`, see https://github.com/impress-org/givewp-next-gen/pull/89/commits/2e4fa0c090dae97e3daac62a0909f84c80d7839f.

File: packages/form-builder/src/settings/donation-goal/index.js [hunk 4]
```diff
@@ -14,23 +14,24 @@ const DonationGoalSettings = () => {
     } = useFormState();
     const dispatch = useFormStateDispatch();
 
+    // @todo Starting with just the Amount Raised goal format.
     const goalFormatOptions = [
         {
             value: 'amount-raised',
```
[the remark this answers is not in the document]
Comment (kjohnson) [reply packages/form-builder/src/settings/donation-goal/index.js:20]: Sure - sounds good. I'll update the naming with the simplification.

[the remark this answers is not in the document]
Comment (kjohnson) [reply packages/form-builder/src/settings/donation-goal/index.js]: Resolved by https://github.com/impress-org/givewp-next-gen/pull/89/commits/7647a2235a58352e023d5a113c2a18e8e24f70b9

[the remark this answers is not in the document]
Comment (kjohnson) [reply packages/form-builder/src/settings/donation-goal/index.js]: Goal Format is now Goal Type, see https://github.com/impress-org/givewp-next-gen/pull/89/commits/ebb48a1670f93a6d6b1b4b10689722f8543a2d42.

**Review by jonwaldstein — APPROVED** [review]
Great job!

**Pull request merged**